airflow/sensors/filesystem.py

```diff
@@ -57,6 +58,9 @@ def poke(self, context):
 
         for path in glob(full_path):
             if os.path.isfile(path):
+                mod_time = os.path.getmtime(path)
+                mod_time = datetime.datetime.fromtimestamp(mod_time).strftime('%Y%m%d%H%M%S')
```

Comment:
```suggestion
                mod_time = datetime.datetime.fromtimestamp(mod_time)
```

otherwise, we will get `20210401160147` (which isn't that readable)

The default is anyways good enough:

```python
>>> str(datetime.datetime.fromtimestamp(os.path.getmtime("some_file.txt")))
'2021-04-01 16:01:47.227895'
```

Reply:
but in SFTP and FTP i using get_mod_time function it's return string format date like this `.strftime('%Y%m%d%H%M%S')`

if you want to readable we should be change shall we change get_mod_time too ?

Reply:
Right, I see what you mean